src/components/pages/lessons/overlay/next-up-overlay.tsx

```diff
@@ -8,13 +8,16 @@ const NextUpOverlay: React.FunctionComponent<{
   url: string
 }> = ({lesson, send, url}) => {
   const {nextLessonTitle, nextUpPath} = useNextUpData(url)
+  const courseImage = lesson?.course?.square_cover_480_url
   return (
     <>
-      <img
-        src={lesson.course.square_cover_480_url}
-        alt={`illustration of ${lesson.course.title} course`}
-        className="w-12 md:w-16 lg:w-32"
-      />
+      {courseImage && (
+        <img
```

Comment:
let's use `Image from next/image`, what do you say?

Reply:
updated